src/layouts/partial-cards.html

```diff
@@ -295,8 +295,8 @@
       return routeMatch ? routeData.view : '';
     }
 
-    computeTitle(views, locationName) {
-      return views && views.length > 0 ? 'Home Assistant' : locationName;
+    computeTitle(locationName) {
+      return locationName ? locationName : 'Home Assistant';
```

Comment:
You want `return locationName || 'Home Assistant';` here to fix the linting problem.